packages/contracts/tests/LenderCommitmentForwarder_Test.sol

```diff
@@ -661,6 +661,155 @@ contract LenderCommitmentForwarder_Test is Testable {
         );
     }
 
+    function test_acceptCommitment_merkle_proof() public {
+        //https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/MerkleProof.sol
+
+        uint256 tokenIdLeaf = 1;
+        bytes32 merkleLeaf = keccak256(abi.encodePacked(tokenIdLeaf)); //  0xc89efdaa54c0f20c7adf612882df0950f5a951637e0307cdcb4c672f298b8bc6;
+        bytes32 merkleRoot = merkleLeaf;
+        bytes32[] memory merkleProof = new bytes32[](0);
```

Comment:
can you create a test with multiple leafs so the test isn't just an empty `bytes32` array

Reply:
There is no need because we do not need to test openzeppelin's merkle proofing library.  A test in this way is a full unit test of our code and a test as you describe will ONLY test openzeppelin's code further.